server/models/User.py

```diff
@@ -88,18 +99,21 @@ class UserSchema(Schema):
     password = fields.String(
         required=True, validate=validate.Length(min=6), load_only=True)
     confirmPassword = fields.String()
+
+    address = fields.String()
+    generalLocation = fields.String()
+
     aboutMe = fields.String()
-    streetAddress = fields.String()
-    city = fields.String()
-    state = fields.String()
-    zipcode = fields.String()
-    country = fields.String()
-    latitude = fields.Float()
-    longitude = fields.Float()
-    formattedAddress = fields.String()
+    chefProfile = fields.String()
```

Comment:
Is this the aboutMe field? It could probably be used for the dual purpose if so

Reply:
I think chefProfile should be separate from aboutMe.  the aboutMe refers to customer profile page.  a chef should be able to have a separate regular  user profile page.